oncotator/Oncotator.py

```diff
@@ -305,6 +320,8 @@ def determineOtherOptions(args):
     opts[OptionConstants.INFER_ONPS] = args.infer_onps
     opts[OptionConstants.CUSTOM_CANONICAL_TX_LIST_FILE] = args.canonical_tx_file
     opts[OptionConstants.COLLAPSE_FILTER_COLS] = args.collapse_filter_cols
+    opts[OptionConstants.REANNOTATE_TCGA_MAF_COLS] = args.prune_tcga_maf_cols
```

Comment:
Should prune_tcga_maf_cols be renamed to reannotate_tcga_maf_calls?  I think it's clearer than prune.